noaa_semantic_search/files.txt

```diff
@@ -0,0 +1,2 @@
+documents.json.bz2
```

Comment:
Do we need this file?

Reply:
let me check, I've checked other workloads and some doesn't have this file

Reply:
you're right that file is not needed. I've pushed commit that removes the file, now integ tests are running

Reply:
test passed, resolving